lib/y2lan_restart_common.pm

```diff
@@ -141,13 +149,18 @@ sub check_network_status {
     else {
         assert_script_run 'dig suse.com|grep \'status: NOERROR\'';    # test if conection and DNS is working
     }
-    assert_script_run 'cat journal.log';                              # print journal.log
-    if ($expected_status eq 'restart') {
-        assert_script_run '[ -s journal.log ]';                       # journal.log size is greater than zero (network restarted)
+    assert_script_run 'cat journal.log';    # print journal.log
+                                            # $check_size is 0 if journal.log is not empty and 1 if it is empty.
+                                            # 1 -> no restart occurred. Checking if expected status was indeed 'no_restart', if not error message is recorded.
+    my $check_size = script_run('[ -s journal.log ]');
```

Comment:
I would save the comments in this case, it would be much more readable to name the variable `my $is_empty` or `my $has_content` because now the size is confusing in the condition, i would expect if has size has content or something like that but feel free to leave it as it. (optional)

Reply:
done